src/align.cpp

```diff
@@ -21,13 +21,13 @@
 static chunk_t *align_var_def_brace(chunk_t *pc, size_t span, size_t *nl_count);
 static chunk_t *align_trailing_comments(chunk_t *start);
 static void align_init_brace(chunk_t *start);
-static void align_func_params();
+static void align_func_params(void);
```

Comment:
@bengardner The reason I did it was just as described in the link you provided.
> In C++, foo() and foo(void) both mean "a function with zero parameters", but some people prefer the second form because it is more explicit.

I prefere the explicit way as it makes clear what the programmer's intention was.
But I understand that people who only program in C++ do not see any use for it. 
So if you prefer the empty parenthesis I am glad with that too. But I would propose to stick to one stile throughout the code.